hotel/config.py

```diff
@@ -0,0 +1,34 @@
+from hotel import *
+
+hotel_config = parse_config(__file__)
+c.include_plugin_config(hotel_config)
```

Comment:
This is how we call our new method for automatically sticking plugin config onto ``c``.  Seems pretty intuitive.